src/main/java/com/google/sps/data/EntryObject.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EntryObject {
+  List<Object> hourList = new ArrayList<>();
+  List<Object> difficultyList = new ArrayList<>();
+  List<String> container = new ArrayList();
+
+  public EntryObject(String hourTitle, String diffTitle) {
+    List<String> container = new ArrayList();
+    container.add(hourTitle);
+    hourList.add(container);
+
+    container.removeAll(container);
+    container.add(hourTitle);
+    difficultyList.add(container);
+  }
+
+  public void addTitle(String title) {
+    List<String> container = new ArrayList();
+    container.add(title);
+    hourList.add(container);
+  }
+
+  public void addHour(String newHour) {
+    List<Integer> container = new ArrayList();
+    container.add(Integer.parseInt(newHour));
+    hourList.add(container);
+  }
+
+  public void addDifficulty(String newDifficulty) {
```

Comment:
On the addThing methods, can you not directly add to hourList/difficultyList?

Reply:
I can't because I need to format it in bracket because the chart API needs a nested list format like : [[item1],[item2]....]